rpm/toggle-monitor/10-keybindings-toggle-monitor

```diff
@@ -0,0 +1,5 @@
+[org/mate/desktop/keybindings/custom10]
+action='/usr/local/bin/monitor-toggle'
```

Comment:
This was forgotten here from the old times when this was installed locally.
```suggestion
action='/usr/bin/monitor-toggle'
```

Reply:
Fixed.